web/html/src/manager/state/recurring-states-list.js

```diff
@@ -0,0 +1,146 @@
+                        </td>
+                        <td className="text-center">{row.scheduleName}</td>
+                        <td className="text-center">{row.cron}</td>
+                        {/*<td className="text-center">{row.createdAt + " " + timezone}</td>*/}
+                        {this.props.disableCreate ? <td className="text-center">{row.targetType}</td> : null}
+                        {/*<td className="text-center">
+                         <input type="checkbox" className="checkbox btn-group" checked={row.skipNext === "true"} onChange={() => {this.props.onSkip(row)}}/>
+                         </td> } */ }
+                        <td className="text-right">
+                            <div className="btn-group">
+                                <Button
+                                    className="btn-default btn-sm"
+                                    title={t("Details")}
+                                    icon="fa-list"
+                                    handler={() => {
+                                        this.props.onSelect(row)
+                                    }}
+                                />
+                                <Button
+                                    className="btn-default btn-sm"
+                                    title={t("Edit")}
+                                    icon="fa-edit"
+                                    handler={() => {
+                                        this.props.onEdit(row)
+                                    }}
+                                />
+                                <ModalButton
+                                    className="btn-default btn-sm"
+                                    title={t("Delete")}
+                                    icon="fa-trash"
+                                    target="delete-modal"
+                                    item={row}
+                                    onClick={i => this.selectToDelete(i)}
+                                />
+                            </div>
+                        </td>
+                    </tr>
+                );
+            });
+        }
+
+        return (
+            <tbody className="table-content">
+            {elements.length > 0 ? elements :
+                <tr>
+                    <td colSpan="4">
+                        <div>{t("No schedules created." + (this.props.disableCreate ? "" : " Use Create to add a schedule."))}</div>
+                    </td>
+                </tr>
+            }
+            </tbody>
+        );
+    };
+
+    render() {
+        const createButton = [
+            <div className="btn-group pull-right">
+                <Button
+                    className="btn-default"
+                    icon="fa-plus"
+                    text={t("Create")}
+                    title="Schedule a new Recurring States Action"
+                    handler={() => {this.props.onActionChanged("create")}}
```

Comment:
Please, remove the `{` wrapping the function call

Reply:
Done.